README.md

```diff
@@ -7,6 +8,10 @@ To know more about Razorpay payment flow and steps involved, please read up here
 
 ## Installation
 
+This has 3 steps: add to project, installation and linking iOS SDK.
```

Comment:
`Linking` is the std term. http://facebook.github.io/react-native/releases/0.36/docs/linking-libraries-ios.html

May be, we can have "Linking IOS sdk" to "Extra steps for linking iOS sdk"